platform/nephos/rules.mk

```diff
@@ -0,0 +1,30 @@
+include $(PLATFORM_PATH)/sdk.mk
+include $(PLATFORM_PATH)/sai.mk
+include $(PLATFORM_PATH)/platform-modules-ingrasys.mk
+include $(PLATFORM_PATH)/docker-orchagent-nephos.mk
+include $(PLATFORM_PATH)/docker-syncd-nephos.mk
+include $(PLATFORM_PATH)/docker-syncd-nephos-rpc.mk
+include $(PLATFORM_PATH)/one-image.mk
+include $(PLATFORM_PATH)/one-aboot.mk
+include $(PLATFORM_PATH)/libsaithrift-dev.mk
+include $(PLATFORM_PATH)/python-saithrift.mk
+include $(PLATFORM_PATH)/docker-ptf-nephos.mk
+
+NPX_DIAG = npx_diag
+$(NPX_DIAG)_URL = "https://github.com/NephosInc/SONiC/raw/master/sdk/npx_diag_2.0.1-20171020"
+
+DSSERVE = dsserve
```

Comment:
is this needed?

Reply:
Yes, this is needed.
Nephos support the npx_diag utility like bcmcmd to show/debug the ASIC information.

Reply:
thanks